### PR TITLE
adds more information about sessions

### PR DESCRIPTION
**Pull request by vedarthk**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -5,4 +5,19 @@ Albeit tiny, this is our attempt to give back to the eco system in some way.
...
+
+The bootcamp will be of two parts of about 3-4 hours each. In first part of the camp we will learn and build a working application with separate frontend and backend. The second part will be about Docker and how can we use it to develop the application and deploy it in production.
+
+**For first session working knowledge of JavaScript is a must. Please read up [https://developer.mozilla.org/en-US/docs/Web/JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript)**
+
+**For second session please install Docker for your operating system from for [here](https://www.docker.com/)**
+
+
+### Building JavaScript Application
+In this session we will learn how a modern web application is developed and deployed. The application will have a backend which will serve data via APIs and frontend will consume the data and present it using UI components. After developing the application we will deploy the application on [Heroku](https://www.heroku.com/) and [Firebase](https://firebase.google.com/) which allows a developer to deploy application without any prior knowledge of DevOps.
```
Comment (rohanBagchi) [inline index.md:19]: After developing the application, we will deploy the backend API on [Heroku](https://www.heroku.com/) and the frontend on [Firebase](https://firebase.google.com/) which allows a developer to deploy application without any prior knowledge of DevOps.

The idea is for us to demonstrate how two independent systems can talk to each other over REST webservices.

**Review by rohanBagchi — APPROVED** [review]
Just update the part in comment. Rest is good to go.

File: index.md (hunk 1 above)
Comment (vedarthk) [reply index.md:19]: @rohanBagchi you can add more information there, I have added a placeholder.

Comment (rohanBagchi) [reply index.md:19]: Ok.

Comment (rohanBagchi) [reply index.md:19]: I will push a commit with the update and then merge. Ok?

Comment (vedarthk) [reply index.md:19]: 👍

Comment (rohanBagchi) [reply index.md:19]: Also, given our plan, I think the google form questionnaire is invalid. Will make it a simple RSVP form.
Also, the date. 29th & 30th of July seems good?

Comment (vedarthk) [reply index.md:19]: SGTM

I think we should have name email and contact number fields in the form where name and email are mandatory ?

Comment (rohanBagchi) [reply index.md:19]: SGTM

**Pull request merged**